Dockerfile-3.9

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-alpine as base
+FROM python:3.9-alpine
```

Comment:
Think you can get rid of this dockerfile entirely and anything else 3.9 related. 

Because this package is versioned nothing will use the 3.13 version until we tell it to, so we don't really need it to support the older python versions.